Sources/TuistCore/Protocols/GeneratorModelLoading.swift

```diff
@@ -17,7 +17,7 @@ public protocol GeneratorModelLoading {
     ///   - path: The absolute path for the project model to load.
     /// - Returns: The Project loaded from the specified path
     /// - Throws: Error encountered during the loading process (e.g. Missing project)
-    func loadProject(at path: AbsolutePath) throws -> Project
+    func loadProject(at path: AbsolutePath, plugins: Plugins) throws -> Project
```

Comment:
will submit a separate PR to remove this protocol as its no longer in use 😅

Reply:
thanks! good point

Reply:
https://github.com/tuist/tuist/pull/2798 will rebase that once your PR lands